test/firebase-query.html

```diff
@@ -353,7 +359,8 @@
             return setFirebaseValue(
                 query.path + '/' + query.data[0].$key + '/val', object.val + 1);
           }).then(function() {
-            expect(domRepeat._instances[0].item).to.be.eql(query.data[0]);
+            var instances = domRepeat._instances || domRepeat.__instances;
```

Comment:
uhhh i think `domRepeat._instances || domRepeat.__instances` is always `domRepeat._instances` so you don't have to or it? :)